DOCS-4813 Error Tracking for Backend Services, Error Tracking Explorer Edits (#17624)
DataDog/documentation: content/en/tracing/error_tracking/explorer.md

---
title: Error Tracking Explorer
kind: documentation
description: Learn about the Error Tracking Explorer.
further_reading:
- link: '/monitors/types/error_tracking'
  tag: 'Documentation'
  text: 'Learn about Error Tracking Monitors'
- link: '/tracing/error_tracking'
  tag: 'Documentation'
  text: 'Learn about Error Tracking for Backend Services'
---

## Overview

> The Error Tracking Explorer shows you a list of ongoing issues. An _issue_ is a group of similar errors that have the same _fingerprint_ and which might need to be fixed depending on how critical it is. 

Reviewer: ```suggestion
The Error Tracking Explorer shows you a list of ongoing issues. An _issue_ is a group of similar errors that have the same _fingerprint_ and which might need to be fixed depending on the severity of the issue. 
```

{{< img src="tracing/error_tracking/error_tracking_explore_inspect.png" alt="Error Tracking Explorer" style="width:100%" >}}

## Explore your issues

Each item listed in the [Error Tracking Explorer][3] surfaces high-level information about the issue, including the following:

-   The error type and the error message.
-   The path to the file in which underlying errors occurred.
-  Information about the issue’s lifetime:
    -   When it was first and last seen.

>     -   Number of error occurrences in the selected time period.
>     -   Graph of error occurrences over time in the selected time period.

Reviewer: ```suggestion
    -   The number of error occurrences in the selected time period.
    -   A graph of error occurrences over time in the selected time period.
```
